README.md

```diff
@@ -187,9 +187,9 @@ The list can be negated to exclude results:
 
 #### `sort`
```

Comment:
You might want to add a paragraph about the chance in the syntax of accessing properties with dot in their name, as the square brackets syntax is new.

Reply:
Done.